M2/Macaulay2/packages/VectorGraphics.m2

```diff
@@ -1,7 +1,7 @@
 -- -*- coding: utf-8 -*-
 newPackage(
         "VectorGraphics",
-        Version => "0.9",
+        Version => "0.91",
```

Comment:
Could you also remove "A package to " from the headline, just so it matches other packages?

Reply:
missed that. so what's the correct formulation? just "produce SVG graphics"? I'm confused

Reply:
I followed the `FirstPackage` template which says `an example Macaulay2 package`...